docker-entrypoint.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -e
 
+# If long & short hostnames are not the same, use long hostnames
+if ! [[ $(hostname) == $(hostname -s) ]]; then
+	export RABBITMQ_USE_LONGNAME=true
+fi
+
+if ! [[ -z ${RABBIT_SSL_CERT_FILE+x} ]] || ! [[ -z ${RABBIT_SSL_KEY_FILE+x} ]] || ! [[ -z ${RABBIT_SSL_CA_FILE+x} ]]; then
```

Comment:
About the `+x` in parameter expansion (eg. `${RABBIT_SSL_CERT_FILE+x}`), Bash man page indicates it should be `:+x` (eg. `${RABBIT_SSL_CERT_FILE:+x}`). I suppose Bash accepts both forms but you should use the documented one.